pkg/rlptools/limit_index.go

```diff
@@ -129,12 +129,23 @@ func (l *LimitIndex) ToLimits() []limitadorv1alpha1.RateLimit {
 	for gwKey, limitsByDomain := range l.gatewayLimits {
 		for domain, limitList := range limitsByDomain {
 			for idx := range limitList {
+				// Currently, Limitador CRD v0.3.1,
+				// the fields "Variables" and "Conditions" are required
+				variables := limitList[idx].Variables
+				if variables == nil {
+					variables = make([]string, 0)
+				}
+				conditions := limitList[idx].Conditions
+				if conditions == nil {
+					conditions = make([]string, 0)
+				}
+
```

Comment:
You might be able to get rid of these 8 lines by using `GetDefaultIfNil` fn from our `common` package.